docs/UserGuide.md

```diff
@@ -241,26 +253,117 @@ An example of an error message is shown below.
+try the command again!
+An example of an error message is shown below.
+![RemarkErrorMsg.png](images/RemarkErrorMsg.png)
+
+### Delete the remark of a company: `unremark` ###
+If the information under remarks is incorrrect or outdated,
+use the `unremark` command to delete it.
+
+**Format:** `unremark INDEX`
+
+>`INDEX`: Deletes remark of the company at INDEX number shown in the displayed company list.
+
+<div class="typography-box tip">
+  <span class="icon">💡</span>
+  <span class="content">
+    <strong>Tip :</strong> To update the remark, you do NOT need to delete using unremark first.
+Directly use the remark command and it overwrites the old remark!
+  </span>
+</div>
+
+**Examples:** </br>
+1. If you wish to delete the remark of Google, type `unremark 1`
+![UnremarkDemo1.png](images/UnremarkDemo1.png)
+
+2. You can view the remark the 1st company as shown below.
+![UnremarkDemo2.png](images/UnremarkDemo2.png)
+
+**Possible Error:**
+
+If you enter an [**invalid index**](#8-what-is-an-invalid-index),
+you will see an error message in the command box. Don’t worry, just edit your index accordingly and
+try the command again!
+An example of an error message is shown below.
+![UnremarkErrorMsg.png](images/UnremarkErrorMsg.png)
+
 ### Edit a company's information: `edit` ###
 If you made a typo or need to update the progress of application,
 use the `edit` command to input the accurate details.
 
-**Format:** `edit INDEX [c/COMPANY_NAME] [n/RECRUITER_NAME] [r/ROLE] [s/APPLICATION_STATUS] [d/DEADLINE] [e/EMAIL] [p/PHONE_NUMBER] [pr/PRIORITY] [nt/NOTE]`
+**Format:** `edit INDEX [c/COMPANY_NAME] [n/RECRUITER_NAME] [r/ROLE] [s/APPLICATION_STATUS] [d/DEADLINE] [e/EMAIL] [p/PHONE_NUMBER] [pr/PRIORITY] [re/REMARK]`
+
+>`INDEX`: Edits the company at INDEX number shown in the displayed company list. </br>
+>`[c/COMPANY_NAME]` etc. : type prefix `c/` followed by new company name to edit company name. </br>
+> At least one of the optional fields must be provided.
 
-* `INDEX`: Edits the company at INDEX number shown in the displayed company list.
-* `[c/COMPANY_NAME]` etc. : type prefix `c/` followed by new company name to edit company name.
-* At least one of the optional fields must be provided.
+<div class="typography-box warning">
+  <span class="icon">⚠️</span>
+  <span class="content">
+    <strong>Warning :</strong> When editing the note, the existing note of the company will be removed i.e adding of note is not cumulative.
+  </span>
+</div>
 
-:warning: When editing the note, the existing note of the company will be removed i.e adding of note is not cumulative.
+<div class="typography-box tip">
+  <span class="icon">💡</span>
+  <span class="content">
+    <strong>Tip :</strong> You can edit remarks through the edit or remark command!
+  </span>
+</div>
 
 **Examples:** </br>
-1.Type `edit 2 s/PI r/Frontend Developer`
+1.You received a email from Microsoft to interview for a Frontend Developer role which is different from the role you applied for.
```

Comment:
received **an** email from .... for a Frontend Developer role **instead**, **which is different from the Software Engineer role you applied for.**